util/user_comparator_wrapper.h

```diff
@@ -58,6 +58,16 @@ class UserComparatorWrapper final : public Comparator {
     return user_comparator_->CanKeysWithDifferentByteContentsBeEqual();
   }
 
+  int CompareTimestamp(const Slice& ts1, const Slice& ts2) const override {
+    return user_comparator_->CompareTimestamp(ts1, ts2);
+  }
+
+  using Comparator::CompareWithoutTimestamp;
+  int CompareWithoutTimestamp(const Slice& a, bool a_has_ts, const Slice& b,
+                              bool b_has_ts) const override {
+    return user_comparator_->CompareWithoutTimestamp(a, a_has_ts, b, b_has_ts);
```

Comment:
Should we bump up counter for user_key_comparison_count as we do in line 33?

Reply:
Yes we should.